### PR TITLE
fix: use cjs as default and lock dep version

### LINKED ISSUE #2603: bug: Can not init logto
### Describe the bug
Run `npm init @logto` and see error:

```
.npm/_npx/75761ac6b328cdec/node_modules/@logto/connector-kit/lib/types.js:4
const language_kit_1 = require("@logto/language-kit");
                       ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/connector-kit/node_modules/@logto/language-kit/lib/index.js from /home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/connector-kit/lib/types.js not supported.
Instead change the require of index.js in /home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/connector-kit/lib/types.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/connector-kit/lib/types.js:4:24)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/connector-kit/lib/index.js:18:20)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/schemas/lib/types/connector.js:4:23)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/schemas/lib/types/index.js:17:14)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/schemas/lib/index.js:32:14)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/cli/lib/commands/database/seed/index.js:9:19)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/cli/lib/commands/install/utils.js:43:16)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/cli/lib/commands/connector/utils.js:21:17)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/cli/lib/commands/connector/add.js:4:17)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/cli/lib/commands/connector/index.js:7:31)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/cli/lib/index.js:10:37)
    at Object.<anonymous> (/home/joey/.npm/_npx/75761ac6b328cdec/node_modules/@logto/cli/bin/logto:2:1) {
  code: 'ERR_REQUIRE_ESM'
}
npm ERR! code 1
npm ERR! path /home/joey/logto
npm ERR! command failed
npm ERR! command sh -c logto-create

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/joey/.npm/_logs/2022-12-06T19_15_53_294Z-debug-0.log
```

### Expected behavior
Logto application created

### How to reproduce?
`npm init @logto`

### Context

OS: 

Environment: 
wsl2 (Ubuntu)

Logto version: 
`@logto/create@1.0.0-beta.16`

Node version:

Postgres version: 

Command of starting Logto:

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

since beta.16 is still using CJS under the hood, must set the package type to cjs and provide mjs in `export`.
fixes https://github.com/logto-io/logto/issues/2603

this is a temp solution. afterwards this repo will be removed to logto-io/logto and become esm based again

## Testing

locally

### CODE REVIEW

File: packages/language-kit/package.json [hunk 1]
```diff
@@ -8,17 +8,21 @@
     "url": "git+https://github.com/logto-io/toolkit.git"
   },
   "license": "MIT",
-  "type": "module",
   "source": "./src/index.ts",
-  "main": "./lib/index.cjs",
-  "exports": "./lib/index.js",
+  "main": "./lib/index.js",
```
Comment (charIeszhao) [inline packages/language-kit/package.json]: Not sure, but should this be `cjs`?

Comment (gao-sun) [reply packages/language-kit/package.json]: no i removed `"type": "module"` now it can be js

Comment (gao-sun) [reply packages/language-kit/package.json]: but you are right cjs should work as well

**Comment by gao-sun** [comment]
this PR is stupid

**Pull request closed without merging**